.travis.yml

```diff
@@ -1,5 +1,18 @@
-sudo: false
+env:
+        - CFLAGS=-I${TRAVIS_BUILD_DIR}/install/include LDFLAGS=-L${TRAVIS_BUILD_DIR}/install/lib LD_LIBRARY_PATH=${TRAVIS_BUILD_DIR}/install/lib
 install: ./autogen.sh
+before_script:
+        - sudo apt-get update
+        - sudo apt-get install libnl-dev
+        # the system provided libibverbs and rdmacm is too old
+        - pushd /tmp && wget https://www.openfabrics.org/downloads/verbs/libibverbs-1.1.8.tar.gz && tar xvfz libibverbs-1.1.8.tar.gz && cd libibverbs-1.1.8 && ./configure --prefix=${TRAVIS_BUILD_DIR}/install && make -j && make install && popd
+        - pushd /tmp && wget https://www.openfabrics.org/downloads/rdmacm/librdmacm-1.0.21.tar.gz && tar xvfz librdmacm-1.0.21.tar.gz && cd librdmacm-1.0.21 && ./configure --prefix=${TRAVIS_BUILD_DIR}/install CC=gcc && make -j && make install && popd
```

Comment:
Use multiple lines for readability, versus one really, really, really, really, really long line